l3embedding/train.py

```diff
@@ -75,7 +75,10 @@ def sample_one_second(audio_data, sampling_frequency, start, label, augment=Fals
     if label:
         start = max(0, int(start * sampling_frequency) - random.randint(0, sampling_frequency))
     else:
-        start = random.randrange(len(audio_data) - sampling_frequency)
+        try:
+            start = random.randrange(len(audio_data) - sampling_frequency)
```

Comment:
This is probably fairly infrequent, but sampling at the closest point before would be a good idea.

Just to clarify, the audio is always at least one second and this case happens when the closest audio sample to the random start time actually causes the end of the window to be a few samples off, is that right?